spread.yaml

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (C) 2015, 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+project: wifi-connect
+
+environment:
+    PROJECT_PATH: /home/wifi-connect
+    TESTSLIB: $PROJECT_PATH/tests/lib
```

Comment:
Though rest of projects in hwe name that folder as "tests", it is true that having at the same time unit and spread ones makes clearer name it "spread_tests". Done.